mount-helper/Makefile

```diff
@@ -1,7 +1,9 @@
 # Makefile to build deb and rpm packages to install mount.ibmshare on target Linux machines.
 
 NAME := mount.ibmshare
```

Comment:
This package has to be removed and continue with new target which worked for IKS as well, we should not maintain two packages